setup.py

```diff
@@ -54,9 +54,9 @@ def run(self) -> None:
     include_package_data=True,
     python_requires=">=3.7, <3.8",
     install_requires=[
-        "python-igraph >= 0.8.3",
+        "python-igraph < 0.8.0",
```

Comment:
The reason for the version bump was that <0.8.3 doesn't build locally for me. It gives me gcc errors up the wazoo

Reply:
weird - with this change all baynet + intervene tests pass locally for me (on your branch oli)